tests/bake.go

```diff
@@ -426,3 +427,9 @@ COPY foo /foo
 	require.Error(t, err, out)
 	require.Contains(t, out, "reading a dockerfile for a remote build invocation is currently not supported")
 }
+
+func testBakeEmpty(t *testing.T, sb integration.Sandbox) {
+	out, err := bakeCmd(sb)
+	require.Contains(t, out, "couldn't find a bake definition")
+	require.Error(t, err, out)
```

Comment:
nit: I would swap these assertions

```suggestion
	require.Error(t, err, out)
	require.Contains(t, out, "couldn't find a bake definition")
```